### ISSUE TITLE
[BUG]  Fix the light BG

### ISSUE THREAD

**Issue by priyankarpal** [Pradumnasaraf/LinkFree-CLI#38]
### What would you like to share?

hii, maintainers please remove the light bg or add dark bg. 

![image](https://user-images.githubusercontent.com/88102392/220941168-bc255450-daba-482a-af92-357e6adae4a1.png)

**Comment by Pradumnasaraf** [comment]
Hey, @priyankarpal 

Thanks for reporting. Yes, you are 100% correct for now we are setting up the background color only we need to add the text color as well because, it may vary from person to person.

Fix

```js
chalk.bgWhite.bold

to

chalk.black.bgWhite.bold // the black will set the color for the text and bgWhite for the background,

```

**Comment by Pradumnasaraf** [comment]
Hey, @priyankarpal again thanks for reporting. It is fixed in `v2.0.1` and also I update the dependencies in LinkFree, you just need to pull changes.
